pkg/model/proto/model.proto

```diff
@@ -717,7 +724,7 @@ message CreateFileShareOpts {
     // The Context
     string context = 12;
     // The Serialized profile
-    string profile = 13;
+    string ProfileId = 13;
```

Comment:
This is profile, not ProfileId.

Reply:
checked with the fileshare team, @PravinRanjan10 this should be ProfileId

Reply:
I discussed with @xing-yang that profile is required in file share operations (create/delete) because vendors may need some customized properties when calling backend drivers to execute the real operation.

Also all protobuf fileds should start with lower case.

Reply:
fixed, will inform the file-share team.